configure

```diff
@@ -632,8 +649,24 @@ cat > $rootdir/config.erl << EOF
 EOF
 
 
+function install_local_rebar(){
+  if [ ! -x "${rootdir}/bin/rebar" ]; then
+      git clone --branch 2.6.0 https://git-wip-us.apache.org/repos/asf/couchdb-rebar.git ${rootdir}/src/rebar
```

Comment:
Good, will add --depth 1.

As for first error, I'm confused, but it seems something wrong happened with clone.